src/Tests/ApiRoutesTest/Properties/launchSettings.json

```diff
@@ -0,0 +1,27 @@
+{
+  "iisSettings": {
+    "windowsAuthentication": false,
+    "anonymousAuthentication": true,
+    "iisExpress": {
+      "applicationUrl": "http://localhost:52434/",
+      "sslPort": 44356
+    }
+  },
+  "profiles": {
+    "IIS Express": {
+      "commandName": "IISExpress",
+      "launchBrowser": true,
+      "environmentVariables": {
+        "ASPNETCORE_ENVIRONMENT": "Development"
+      }
+    },
+    "ApiRoutesTest": {
```

Comment:
Does this remove Kestrel?

Reply:
> Does this remove Kestrel?

This is just a xUnit Test Project, You can remove all of this config.
Also, it is possible to refactor the code, By deleting the ApiRouteTest project and implement it inside of the SERVER project as a new endpoint "/graph"
But this approach is better as the author said in the article :
https://andrewlock.net/adding-an-endpoint-graph-to-your-aspnetcore-application/